### PR TITLE
TRUNK-248 Add JUnit @should annotations according to code comments

### PR DESCRIPTION
**Pull request by hilz041**
TRUNK-248 Add JUnit @should annotations according to code comments

## Description
I have added @should annotations on methods that need  JUnit tests as follows,
@should return criteria if containingAnyFormField is not empty.
@should return a list of patientIdentifiers when patientIdentifierTypes and patient is not null 
@should return criterion of given identifierTypes 

## Related Issue

see https://issues.openmrs.org/browse/TRUNK-248

## Checklist:

- [x ] My pull request only contains one single commit.
- [x ] My pull request is based on the latest master branch
  `git pull --rebase upstream master`.
- [ x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.
- [x ] My code follows the code style of this project.
- [ x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x ] All new and existing tests passed.

### CODE REVIEW

**Comment by hilz041** [comment]
Add @should annotations according to code comments.

**Comment by dkayiwa** [comment]
@hilz041 after creating a pull request, do you take time to look at the number of commits contained in it? :)

File: api/src/main/java/org/openmrs/api/db/hibernate/HibernatePatientDAO.java [hunk 1]
```diff
@@ -268,11 +270,17 @@ public void deletePatientIdentifierType(PatientIdentifierType patientIdentifierT
...
 		}
 		
-		// TODO add junit test for getting by patients
+
+		
 		if (!patients.isEmpty()) {
 			criteria.add(Restrictions.in("patient", patients));
 		}
 		
+		if (patients.size() > 0) {
```
Comment (teleivo) [inline api/src/main/java/org/openmrs/api/db/hibernate/HibernatePatientDAO.java]: @hilz041 you need to remove this block. it is a duplicate of the block above.

**Comment by teleivo** [comment]
@hilz041 please address my comment to your code and squash your commits :)

File: api/src/main/java/org/openmrs/api/db/hibernate/HibernatePatientDAO.java (hunk 1 above)
Comment (hilz041) [inline api/src/main/java/org/openmrs/api/db/hibernate/HibernatePatientDAO.java]: let me address it . @teleivo

**Comment by hilz041** [comment]
@dkayiwa  every time i pull i see other earlier pulls, how can i delete these pulls so that i start from scratch ?

**Comment by dkayiwa** [comment]
git checkout master
git pull upstream master
git checkout branchname
Put in your changes and create a pull request.

**Pull request closed without merging**